src/i18n.ts

```diff
@@ -0,0 +1,18 @@
+import i18next from "i18next";
+import { initReactI18next } from "react-i18next";
+import en from "./locales/en.json";
+
+export const defaultNS = "ns1";
```

Comment:
With only one localization file, you could use the default react-i18next namespace, which is `translation`.